web/public/mgmt/config.yml

```diff
@@ -2522,6 +2522,54 @@ collections:
           - { label: "school/course", value: "school-course" }
           - { label: "object or vehicle", value: "object-vehicle" }
 
+  - name: "raffle-bingo-steps"
+    label: "Raffle Bingo Steps (Navigator with Webflow mappings)"
+    folder: "content/src/roadmaps/raffle-bingo-steps"
+    summary: "{{displayname}} - {{name}}"
+    identifier_field: "{{filename}}"
+    slug: "{{fields.filename}}"
+    delete: true
+    sortable_fields: ["id", "name", "{{urlSlug}}", "industryId"]
+    create: true
+    editor:
+      preview: true
+    fields:
```

Comment:
[boulder] Take another look at these fields. It looks like we aren't representing the collection items correctly (Ex: body is missing, we don't need license name, etc)

Reply:
@AnJuHyppolite this is still missing.

Reply:
[boulder] @AnJuHyppolite body is still missing from the CMS.

Reply:
I see `body` is now included. Marking resolved.